components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -1,108 +1,176 @@
 package mozilla.components.browser.errorpages
 
 import android.content.Context
+import android.support.annotation.RawRes
 import android.support.annotation.StringRes
-import android.webkit.WebViewClient
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object ErrorPages {
+
     /**
      * Enum containing all supported error types that we can display an error page for.
      */
     enum class ErrorType(
```

Comment:
Since this `ErrorType` enum has gotten pretty large now that we've merged all the error codes together, maybe we could make this a top-level class in the file or move it to a different file? I have no strong preference between either option, but I do think it'll make it easier to read the class if we moved the enum somewhere else - that's just me though so feel free to disagree. :)